manifests/init.pp

```diff
@@ -109,6 +109,7 @@
     enable_puppet         => true,
     enable_docker         => true,
     num_workers           => $num_pulp_workers,
+    is_parent             => true,
```

Comment:
Yup, that's it. Only need the params.pp in the case you have in the other PR, when adding it to the class's parameters list.